### PR TITLE
QDB-9666 - Provide an env variable for QDB_API_PATH.

### PR DESCRIPTION
**Pull request by vianneyPL**

### CODE REVIEW

**Review by mkurdej — APPROVED** [review]
LGTM. Update the doc?

File: quasardb/CMakeLists.txt [hunk 1]
```diff
@@ -11,7 +11,13 @@ set(PACKAGE_NAME quasardb)
 # )
 
 # step 1: discover libraries
-set(QDB_API_DIR "${CMAKE_SOURCE_DIR}/../qdb")
+if(DEFINED ENV{QDB_API_PATH})
+  message("Getting qdb api from $ENV{QDB_API_PATH}")
```
Comment (mkurdej) [inline quasardb/CMakeLists.txt:15]: Nit: `message(STATUS`

Comment (mkurdej) [inline quasardb/CMakeLists.txt:15]:
```suggestion
  message(STATUS "Getting qdb API from environment variable QDB_API_PATH: $ENV{QDB_API_PATH}")
```

Comment (vianneyPL) [reply quasardb/CMakeLists.txt]: done.

Comment (vianneyPL) [reply quasardb/CMakeLists.txt]: done.

**Pull request merged**